docs/source/user/getting_started/advanced_workflow_concepts.rst

```diff
@@ -0,0 +1,308 @@
+resources, and will run the command which matches those resources.
+
+There are lots of :ref:`resource options <reference/_autosummary/matflow.ResourceSpec:matflow.ResourceSpec>`
+available that can be requested.
+
+Scheduler arguments can be passed like this e.g. to target high memory nodes:
+
+.. code-block:: yaml
+
+    resources:
+    any:
+        num_cores: 10
+        SGE_parallel_env: smp.pe
+        scheduler_args:
+        options:
+            -l: mem512
+
+
+If you have set resource options at the top level (for the whole workflow), but would like to "unset" them for a particular task,
+
+you can pass an empty dictionary:
+
+.. code-block:: yaml
+
+    - schema: simulate_VE_loading_damask
+    resources:
+        main:
+        num_cores: 16
+        scheduler_args:
+            options: {} # "Clear" any previous options which have been set.
+    inputs:
+
+
+Task sequences
+----------------
+
+Matflow can run tasks over a set of inputs values, if the iterations are independent.
+For this, you use a ``sequence``, and a ``nesting_order`` to control the nesting of the loops
+but you can also "zip" two or more lists of inputs by using the same level of nesting.
+Lower values of ``nesting_order`` act like the "outer" loop.
+
+.. code-block:: yaml
+
+    tasks:
+    - schema: my_schema
+    sequences:
+    - path: inputs.conductance_value
+    values:
+    - 0
+    - 100
+    - 200
+    nesting_order: 0
+
+Groups
+-------
+
+To combine outputs from multiple sequences, you can use a ``group`` in a task schema:
+
+.. code-block:: yaml
+
+    - objective: my_task_schema
+    inputs:
+    - parameter: p2
+        group: my_group
+
+combined with a ``groups`` entry in the task itself.
+
+.. code-block:: yaml
+
+    - schema: my_task_schema
+      groups:
+    - name: my_group
+
+
+Then whichever parameters are linked with the group in the task schema will be received by the task as a list.
+
+
+Here is an example workflow using sequences and groups that you might wish to run to solidify your understanding
+
+.. code-block:: yaml
+
+    # groups_workflow.yaml
+
+    template_components:
+      task_schemas:
+        - objective: s1
+          inputs:
+            - parameter: p1
+          outputs:
+            - parameter: p2
+          actions:
+            - commands:
+                - command: echo $(( <<parameter:p1>> + 1 )) # This is printed to stdout
+                - command: echo $(( <<parameter:p1>> + 1 )) # This is captured as p2
+                  stdout: <<int(parameter:p2)>>
+        - objective: s2
+          inputs:
+            - parameter: p2
+              group: my_group
+          outputs:
+            - parameter: p3
+          actions:
+            - commands:
+                - command: echo <<parameter:p2>> # This one is printed to stdout
+                - command: echo $(( <<sum(parameter:p2)>> )) # This is captured as p3
+                  stdout: <<parameter:p3>>
```

Comment:
Maybe wrap this `stdout` in `int()` as well, for consistency?

Reply:
Fair point. Is `int()` a MatFlow syntax? Or some python wrapping that MatFlow does? Either way if it's not obvious to me, it probably needs explaining (or just removing in both examples?)

Reply:
It's a Python-inspired hpcflow syntax for casting output strings from shell commands to other types (see: https://github.com/hpcflow/hpcflow-new/blob/develop/hpcflow/sdk/core/commands.py#L369)